app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -394,6 +394,10 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
       if ($scope.emsCommonModel.emstype === 'openstack') {
         $scope.emsCommonModel.tenant_mapping_enabled = false;
       }
+    } else if ($scope.emsCommonModel.emstype === 'nuage_network') {
+      $scope.emsCommonModel.default_api_port = $scope.getDefaultApiPort($scope.emsCommonModel.emstype);
+      $scope.emsCommonModel.event_stream_selection = "none";
```

Comment:
Please use single quotes `'none'` here as well. I see we're following a pattern here, but we shouldn't IMO.

Reply:
Fixed!